bloom-housing/bloom
# feat: passwordless frontend

# Pull Request Template

## Issue Overview

This PR addresses #3892


## Description

Includes just the frontend for the sign-in screen and the verify screen.

## How Can This Be Tested/Reviewed?

This will just be a visual QA. Designs are [here](https://www.figma.com/file/2LSD4B4yBIxAUGbZQCVOCv/Passwordless?type=design&node-id=2888-8762&mode=design&t=860jUbwRGdiXMpZ0-0).

Turn on the passwordless environment variable to view the sign in page. You will be able to toggle between the code and password versions.

Visit `/verify` to view the verify page. The alert copy and links included are based on the create account screens (the sign-in screens have different alert copy and links). Including the differences between the sign-in and create account versions will be added when the backend is complete.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
